irc/nickserv.go

```diff
@@ -1049,6 +1122,7 @@ func nsPasswdHandler(service *ircService, server *Server, client *Client, comman
 			if !nsLoginThrottleCheck(service, client, rb) {
 				return
 			}
+			errorMessage = nsConfirmPassword(server, target, params[0])
 			accountData, err := server.accounts.LoadAccount(target)
```

Comment:
Did you mean to delete this logic since it was moved into a function?

Reply:
Wow, yes, this branch is terrible

Reply:
There's one difference where newPassword is replaced with "" if it's "*".